CDCgov/prime-reportstream
# Updated valuesets to include additional specimen source sites from SR

This PR ...

Adds additional specimen sources to the  valueset

Test Steps:
1. Have Covid pipeline settings for RI
2. Send the two attached files through the pipeline with Postman
3. Verify SPM-8 shows the the descriptions etc. for the two new values.

## Changes
- Added two new values we will be receiving from SimpleReport to the specimen_source_site valueset

## Checklist

### Testing

## Linked Issues
- Fixes #12433
